package.json

```diff
@@ -19,7 +19,9 @@
   },
   "devDependencies": {
     "code": "^1.5.0",
+    "eslint": "^1.8.0",
     "eslint-config-nodesecurity": "^1.1.0",
+    "eslint-plugin-hapi": "^2.0.0",
```

Comment:
Ah it probably only worked on my end because I'm on npm@3, so that makes sense. Yeah let's keep that there then. Thanks for checking it out!